lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -152,12 +163,30 @@ class VersionSectionSelector extends Component<SelectorProps> {
         }
       })
     }
+
     const text = hasCriticalError
       ? 'critical error'
       : validationSummary.errorCount
+
+    let diffLabel
+    if (comparedVersion && comparedVersionIndex) {
+      const diff = validationSummary.errorCount - comparedVersion.validationSummary.errorCount
+      const diffClassName = diff > 0 ? 'label-danger' : (diff < 0 ? 'label-success' : 'label-default')
+
+      diffLabel = (
+        <span
+          className={diffClassName}
+          style={{border: '0.5px solid #ccc'}}
+          title={`Difference compared to Version ${comparedVersionIndex}`}
+        >
+          {formatDelta(diff)}
+        </span>
+      )
+    }
+
     return (
       <BsLabel className={`gtfs-error-${highestPriority}-label-solid`}>
-        {text}
+        {text} {diffLabel}
```

Comment:
I'm not too crazy about the label within a label. I think perhaps the diff label should be adjacent to the other label (rather than nested within it).

Reply:
Here is a screenshot comparing what we can do:

![image](https://user-images.githubusercontent.com/56846598/100800708-f4fa1900-33f4-11eb-942a-3e10e5e760dc.png)

Reply:
The third one has my preference now.